packages/protocol/test/governance/governance.ts

```diff
@@ -458,6 +480,118 @@ contract('Governance', (accounts: string[]) => {
     })
   })
 
+  describe('#setParticipationFloor', () => {
+    const differentParticipationFloor = toFixed(2 / 100)
+
+    it('should set the participation floor', async () => {
+      await governance.setParticipationFloor(differentParticipationFloor)
+      const [, actualParticipationFloor, ,] = await governance.getParticipationParameters()
+      assertEqualBN(actualParticipationFloor, differentParticipationFloor)
+    })
+
+    it('should emit the ParticipationFloorSet event', async () => {
+      const resp = await governance.setParticipationFloor(differentParticipationFloor)
+      assert.equal(resp.logs.length, 1)
+      const log = resp.logs[0]
+      assertLogMatches2(log, {
+        event: 'ParticipationFloorSet',
+        args: {
+          participationFloor: differentParticipationFloor,
+        },
+      })
+    })
+
+    it('should revert if new participation floor is below 0', async () => {
+      await assertRevert(governance.setParticipationFloor(toFixed(-1 / 100)))
```

Comment:
This isn't really a thing since the parameter is passed in as an unsigned integer.